drivers/hmis/app/graphql/types/base_object.rb

```diff
@@ -49,6 +49,18 @@ def load_ar_association(object, association, scope: nil)
       dataloader.with(Sources::ActiveRecordAssociation, association, scope).load(object)
     end
 
+    def load_ar_scope(scope:, id:)
+      dataloader.with(Sources::ActiveRecordScope, scope).load(id)
+    end
+
+    def load_last_user_from_versions(object)
+      versions = load_ar_association(object, :versions)
+      last_user_id = versions.max_by(&:created_at)&.whodunnit
+      return unless last_user_id && last_user_id =~ /\A[0-9]+\z/
+
+      load_ar_scope(scope: Hmis::User.all, id: last_user_id)
```

Comment:
Perhaps this should be `Hmis::User.viewable_by(current_user)` however that requires that the user has `can_impersonate_users` which is maybe not right. Perhaps we should adjust Hmis;:User.viewable_by ?

Reply:
In this case, the scope should be `Hmis::User.all` because there are no restrictions on seeing who last touched a record that you can see.

The `viewable_by` scope will need to be adjusted to remove the "can impersonate" limit, I think. That scope is used to determine which users show up on the User list on the HMIS Admin dashboard. "viewable by" probably isn't the right term for that scope, since it won't be user-dependent. It should probably just be called `hmis_users` or something. The change feels unrelated to this PR though.

Reply:
Do we need to include_deleted?  Hmis::User is acts_as_paranoid, so if a user was deleted, we should probably still know they changed something.

Reply:
Good call, yes we should include deleted

Reply:
Good catch, I'm including deleted users now